src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -321,11 +323,13 @@ PARAM_DEFINE_FLOAT(MC_ROLLRATE_MAX, 220.0f);
 /**
  * Max pitch rate
  *
- * Limit for pitch rate, has effect for large rotations in autonomous mode, to avoid large control output and mixer saturation.
+ * Limit for pitch rate in manual and auto modes (except acro).
+ * Has effect for large rotations in autonomous mode, to avoid large control
+ * output and mixer saturation.
  *
  * @unit deg/s
  * @min 0.0
- * @max 360.0
+ * @max 3600.0
```

Comment:
Should be 1800, as 2000 is the hardware limit. We should bound it to what we can still sense.